docs/basics/integrations/webhooks.md

```diff
@@ -0,0 +1,78 @@
+---
+id: webhooks
+title: Webhooks Integration
+sidebar_label: Webhooks
+description: Setup a webhook to which Sauce Labs test results are automatically pushed.
+keywords:
+- get-test-results
+- webhooks
+- how-to
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<p><span className="sauceDBlue">COMING SOON</span></p>
+
+The Sauce Labs Webhooks integration allows you specify a URL to which Sauce Labs will push test result data as it becomes available, rather than forcing you to constantly poll for updates.
+
+## What You'll Need
+
+* A Sauce Labs account (if you don't have one, start a [free trial](https://saucelabs.com/sign-up))
+* Your Sauce Labs [Username and Access Key](https://app.saucelabs.com/user-settings)
+* A URL able to consume events data pushed from Sauce Labs
+
+
+## Configure a Webhook
+
+1. From your Sauce Labs account, navigate to the [Account Integrations](https://app.staging.saucelabs.net/integrations) page.
```

Comment:
Is it `Account Integrations`? Shouldn't it be just `Integrations`?

Reply:
Well...the page itself is just "Integrations", but you get to it from the "Account" drop-down menu on the main screen of Sauce Labs app, so I feel like this is accurate in terms of navigation.